index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="fonts/css/all.css">
+  <title>Elena</title>
+</head>
+<body>
+    <div id="wrapper">
+      <div id="header">
+        <div class="container">
+          <div class="logo">
+            <a href="#"><img src="img/logo.png" alt="Logo"></a>
+          </div>
+          <div class="nav">
+            <input type="checkbox" name="toggle" id="menu" class="toggleMenu">
+            <label for="menu" class="toggleMenu"><i class="fa fa-bars"></i>Menu</label>
+            <ul>
+              <li><a href="#">HOME</a></li>
+              <li><a href="#">SERVICES</a></li>
+              <li><a href="#">Features</a></li>
+              <li><a href="#">CONTACT</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div id="home" name="home">
+        <div class="container">
+          <h1>HAVE A GOOD DAY</h1>
+          <h3 class="text-line">JUST FOR YOU ON THEMEFOREST</h3>
+          <a href="#" class="btn">GET YOUR COPY NOW</a>
+        </div>
+      </div>
+      <div id="servises" class="box">
+        <div class="container">
+          <div class="servises-wrap">
+            <div class="servises-item profesionals">
+              <h5 class="title">
+                PROFESIONALS
+              </h5>
+              <p>Lorem ipsum dolor sit amet consectetur elit. </p>
```

Comment:
`sit amet` -- unknown character.
` elit. </p>` -- unnecessary space before the closing tag.